recipe/meta.yaml

```diff
@@ -20,25 +32,33 @@ outputs:
       run_exports:
         strong:
           - cudatoolkit {{ cuda_compiler_version }}|{{ cuda_compiler_version }}.*
-          - __glibc >=2.17  # [linux and cuda_compiler_version == "11.0"]
+{% if cuda_major_minor >= (11, 0) %}
+          - __glibc >=2.17        # [linux]
+{% endif %}
     requirements:
       build:      # [win]
         - m2-sed  # [win]
       host:
         # Needed to symlink libcuda into sysroot libs.
         - {{ compiler("c") }}
-        - sysroot_linux-64 2.17  # [linux and cuda_compiler_version == "11.0"]
-        - __glibc >=2.17         # [linux and cuda_compiler_version == "11.0"]
+{% if cuda_major_minor >= (11, 0) %}
+        - sysroot_linux-64 2.17   # [linux]
+        - __glibc >=2.17          # [linux]
+{% endif %}
       run:
-        - sed                    # [linux]
-        - sysroot_linux-64 2.17  # [linux and cuda_compiler_version == "11.0"]
-        - __glibc >=2.17         # [linux and cuda_compiler_version == "11.0"]
+        - sed                     # [linux]
+{% if cuda_major_minor >= (11, 0) %}
+        - sysroot_linux-64 2.17   # [linux]
+        - __glibc >=2.17          # [linux]
+{% endif %}
```

Comment:
In cuDNN/cuTENSOR feedstocks, we put `__glibc` in `run_constrained`, not `run`, though I feel the effect should be the same given that you also added `run_exports`?
https://github.com/conda-forge/cutensor-feedstock/blob/cab81ab8c0ab8c3f4d37d05b6883094b3510585e/recipe/meta.yaml#L56-L58

Reply:
The restraints [were already configured like this in the recipe](https://github.com/conda-forge/nvcc-feedstock/blob/master/recipe/meta.yaml#L12-L35). I only changed from selector syntax to Jinja conditionals because I found no other way of say `cuda_major >= 11` (see commits at #50 for a collection of failed attempts).

Reply:
Yeah I did notice the constraints were added in `run_exports`. What I meant is it seems conda expects `__glibc`, which is a virtual package, to be added in `run_constrained`. At least this is what's documented:
https://docs.conda.io/projects/conda-build/en/latest/resources/define-metadata.html#run-constrained
It seems there might be some subtitles I do not fully understand 😅